sympy/series/sequences.py

```diff
@@ -721,9 +722,9 @@ class SeqAdd(SeqExprOp):
 
     Rules:
         * The interval on which sequence is defined is the intersection
-        of respective intervals of sequences.
-        * Anything + EmptySequence, remains unchanged
-        * Other rules are defined in _add methods of sequence classes
+            of respective intervals of sequences.
+        * Anything + :class:`EmptySequence` remains unchanged
+        * Other rules are defined in ``_add`` methods of sequence classes
 
```

Comment:
Same as [above](https://github.com/sympy/sympy/pull/9590/files#r34171747), there are few more such cases not visible in diff.